crates/task-impls/src/request.rs

```diff
@@ -181,6 +190,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, Ver: StaticVersionType + 'st
             .whole_committee(view)
             .into_iter()
             .collect();
+        let leader = self.da_membership.leader(view);
```

Comment:
Is this always supposed to be the DA leader? Looks like when handling the `QuorumProposalMissing` event, we also call this function, so I'm wondering in that case whether we should use the quorum leader.

Reply:
it really should be the quorum leader, but for now it's always the same I think we should actually combine the memberships in the long term since right now they are references to the same thing